tests/e2e/snapshotrestore/Vagrantfile

```diff
@@ -0,0 +1,145 @@
+ENV['VAGRANT_NO_PARALLEL'] = 'no'
+NODE_ROLES = (ENV['E2E_NODE_ROLES'] ||
+  ["server-0", "server-1", "server-2", "agent-0", "agent-1"])
+NODE_BOXES = (ENV['E2E_NODE_BOXES'] ||
+  ['generic/ubuntu2004', 'generic/ubuntu2004', 'generic/ubuntu2004', 'generic/ubuntu2004', 'generic/ubuntu2004'])
+GITHUB_BRANCH = (ENV['E2E_GITHUB_BRANCH'] || "master")
+RELEASE_VERSION = (ENV['E2E_RELEASE_VERSION'] || "")
+EXTERNAL_DB = (ENV['E2E_EXTERNAL_DB'] || "etcd")
+HARDENED = (ENV['E2E_HARDENED'] || "")
+REGISTRY = (ENV['E2E_REGISTRY'] || "")
+RANCHER = (ENV['E2E_RANCHER'] || "")
```

Comment:
No need to support any of these Variables for this test, this is an etcd only test that does not deploy any workloads.
So EXTERNAL_DB = "etcd" should be the only option.